src/network/networkd-route.c

```diff
@@ -55,7 +55,7 @@ static const char * const route_type_table[__RTN_MAX] = {
 };
 
 assert_cc(__RTN_MAX <= UCHAR_MAX);
-DEFINE_PRIVATE_STRING_TABLE_LOOKUP(route_type, int);
+DEFINE_STRING_TABLE_LOOKUP(route_type, int);
```

Comment:
Please do not reuse the table. But introduce new table. E.g., FR_ACT_BLACKHOLE -> "blackhole", as some values are not used in fib rules.